README.md

```diff
@@ -116,18 +116,120 @@ One of the big concerns with IPFS, and even cloud data storage in general is enc
 
 ## Usage and Features
 
-### Spinning up a Node
+Before attempting to use Temporal you will need to install it. Even if you are going to be using our dockerized tooling, an install of Temporal is needed primarily for configuration file initialization.
 
-Once you have a `config.json` set up (a template can be generated using `temporal init`), you can run the following commands to use docker-compose to spin up Temporal:
+Please note that a full-blown Temporal instance including the payment processing backend can take awhile, and requires an API key for [ChainRider](https://chainrider.io/) as well as a fully synced [geth node](https://github.com/ethereum/go-ethereum), and [bchd node](https://github.com/gcash/bchd). We will *not* be covering the setup of either chainrider, geth, and bchd, please consult appropriate documentation for setting those up. Should you want to read about our payment processing backedn see [RTradeLtd/Pay](https://github.com/RTradeLtd/Pay)
```

Comment:
For a minimal client for IPFS (no payments baked in, potentially only a light ethereum client integration, and no BCH) what do you need to do? Is it even possible to configure this?

Reply:
> For a minimal client for IPFS (no payments baked in) what do you need to do?

I updated the readme indicating that for minimal client for IPFS you dont need to do anything

As for the light client stuff as mentioned on our telegram conversation it is definitely possible.